director/director_test.go

```diff
@@ -677,6 +683,47 @@ func TestDirectorRegistration(t *testing.T) {
 		teardown()
 	})
 
+	t.Run("origin-storage-type-and-disable-test", func(t *testing.T) {
+		c, r, w := setupContext()
```

Comment:
Can you also create a test that is a posix storage type but tests are disabled an ensure that tests aren't run?